test/data/schemas.test.ts

```diff
@@ -233,11 +233,23 @@ const isURLValid = (url: string): boolean => {
 test('all programs have valid URLs', async tap => {
   for (const [programId, data] of Object.entries(PROGRAMS)) {
     for (const url of Object.values(data.url ?? [])) {
-      tap.ok(isURLValid(url), `${programId} has invalid URL`);
+      if (typeof url === 'string') {
+        tap.ok(isURLValid(url), `${programId} has invalid URL`);
+      } else {
+        tap.ok(false, `${programId} has invalid URL`);
+      }
     }
   }
 });
 
+test('program JSON files match schemas', async tap => {
+  const ajv = new Ajv();
```

Comment:
@oyamauchi I added this test that tests all programs using the program schema. Let me know if you think this should be done on a state by state basis instead, but this matches how the programs are loaded by the application.